corehq/apps/indicators/templates/indicators/forms/copy_to_domain.html

```diff
@@ -5,24 +5,14 @@
 
 {% block title %}{% trans 'Bulk Copy Indicators' %}{% endblock title %}
```

Comment:
@biyeun Do you know if this page is still used? It looks MVP-related and I didn't find it linked anywhere.

Reply:
my guess is this is dead but will let B confirm